include/ftl/config.h

```diff
@@ -26,43 +26,43 @@
 
 // Determine the OS
 #if defined(WIN32) || defined(_WIN32) || defined(__WIN32)
-	#define FTL_OS_WINDOWS
```

Comment:
I'm not super thrilled with this format. I could live with it, but I would prefer the old style. `#` on the first column is a remnant of super super old preprocessors. All modern preprocessors allow non column zero defines and macros

Reply:
I'm not really either, but either non-indented or indented after the `#` are the only choices.